src/controllers/dlnaprofile.js

```diff
@@ -476,11 +476,11 @@ define(["jQuery", "loading", "fnchecked", "emby-select", "emby-button", "emby-in
 
             html += "<div>";
             html += '<a is="emby-linkbutton" href="#" class="lnkEditSubProfile" data-profileindex="' + i + '">';
-            html += "<p>" + Globalize.translate("ValueCodec").replace("{0}", profile.Codec || allText) + "</p>";
```

Comment:
I would keep these changes and revert the global variable removal for now because you can't be expected to fix all those linting errors at the same time. Hopefully you can just revert to just before the variable change.